systemverilog-plugin/uhdmcommonfrontend.cc

```diff
@@ -120,6 +120,10 @@ void UhdmCommonFrontend::execute(std::istream *&f, std::string filename, std::ve
             this->shared.link = true;
             // Surelog needs it in the command line to link correctly
             unhandled_args.push_back(args[i]);
+        } else if (args[i] == "-formal") {
```

Comment:
We could add this new switch to be printed out by _print_read_options_, [line 33](https://github.com/chipsalliance/yosys-f4pga-plugins/blob/6b61eb85b9042aabb4e32d57d37891fa846e8815/systemverilog-plugin/uhdmcommonfrontend.cc#L33).

Reply:
Added (and fixed some missing newlines).